model/resource/process.yaml

```diff
@@ -8,6 +8,7 @@ groups:
       - ref: process.pid
       - ref: process.parent_pid
       - ref: process.executable.name
+      - ref: process.owner
```

Comment:
oops, this change resulted in changing  `requirement_level` on `process.executable.name`.